src/helpers.ts

```diff
@@ -254,3 +257,82 @@ export function getEncodedData(commands: string[], state: string[]): string {
   ]);
   return weirollWalletInterface.encodeFunctionData('executeWeiroll', [commands, state]);
 }
+
+export function getVerificationHash(script: WeirollScript, receiptToken: AddressArg, inputTokens: AddressArg[]) {
+  // TODO: confirm token order for encoding hash
```

Comment:
It definitely looks right, isn't it?

Reply:
I can confirm that the hash is correct when there are no tokens present. i'm still concerned that the hashing function needs some sorting for the input tokens if there are multiple. its not clear to me whether they are set in a specific order or its just arbitrary. if its set in a specific order we can sort them, otherwise we need to do a lookup and check how they are ordered on-chain